docs/troubleshooting.md

```diff
@@ -0,0 +1,127 @@
+community-operators   appregistry   https://quay.io/cnr   community-operators   Community Operators   Red Hat     Succeeded   The object has been successfully reconciled   22h
+redhat-operators      appregistry   https://quay.io/cnr   redhat-operators      Red Hat Operators     Red Hat     Succeeded   The object has been successfully reconciled   22h
+```
+
+If the status of any of the OperatorSource is anything other than `Succeeded`, you can investigate the OperatorSource with 
+`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-download-from-datastore) of the doc also contains helpful information about debugging an OperatorSource. 
+
+If the OperatorSources look healthy, check that the CatalogSourceConfigs for the corresponding OperatorSources were created.
+
+```
+$ oc get csc -n openshift-marketplace
+
+NAME                  STATUS      MESSAGE                                       AGE
+certified-operators   Succeeded   The object has been successfully reconciled   23h
+community-operators   Succeeded   The object has been successfully reconciled   23h
+redhat-operators      Succeeded   The object has been successfully reconciled   23h
+```
+
+If the status of any of the CatalogSourceConfigs is anything other than `Succeeded`, you can inspect the CatalogSourceConfig with `oc describe csc <Name-of-CatalogSourceConfig>`. [This section](#catalogsourceconfig-stuck-in-configuring-phase) of the doc also contains helpful information about debugging a CatalogSourceConfig. 
+
+If the CatalogSourceConfigs look healthy, make sure that there's a pod corresponding to each CatalogSourceConfig and that they are in the `Running` state.
+
+```
+$ oc get pods -n openshift-marketplace
+
+NAME                                    READY     STATUS    RESTARTS   AGE
+certified-operators-79f876c76d-mbtjg    1/1       Running   0          86m
+community-operators-854c564f5f-9vk2x    1/1       Running   0          86m
+marketplace-operator-6dcd66cb8c-2wmkz   1/1       Running   0          86m
+redhat-operators-647fd668fc-9ff5d       1/1       Running   0          86m
+
+```
+
+If any pod is in a state other than `Running` inspect the logs of the pod with `oc logs <Name-of-Pod>`.
+
+If all pods look healthy, make sure that there's a CatalogSource for each corresponding CatalogSourceConfigs.
+
+```
+$ oc get catalogsource -n openshift-marketplace 
+
+NAME                  NAME                  TYPE      PUBLISHER   AGE
+certified-operators   Certified Operators   grpc      Red Hat     23h
+community-operators   Community Operators   grpc      Red Hat     23h
+redhat-operators      Red Hat Operators     grpc      Red Hat     23h
+
+```
+If you need help debugging the CatalogSources, please reach out to the #forum-operators channel in the [upstream Kubernetes slack instance](https://slack.k8s.io/) or the [google group for Operator-Framework.](https://groups.google.com/forum/#!forum/operator-framework). 
```

Comment:
I suggest adding a separate section called "Where to go for help` or something like that and add this line there instead of repeating this in multiple places.

I also think you can link to the slack channel itself.